sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -272,7 +277,7 @@ public partial class DataCollectionRuleAssociationData : Azure.ResourceManager.M
         public Azure.Core.ResourceIdentifier DataCollectionRuleId { get { throw null; } set { } }
         public string Description { get { throw null; } set { } }
         public Azure.ETag? ETag { get { throw null; } }
-        public string MetadataProvisionedBy { get { throw null; } }
```

Comment:
Here is another breaking change similar with the one you have mitigated in this commit: https://github.com/Azure/azure-sdk-for-net/pull/35018/commits/8aee5bf0e287ceb74e814fe26d6b9d8f2975a347
Please apply the same solution to this property as well.